src/docs/sphinx/advancedExamples/validationStudies/deviatedPoroElasticWellbore/Example.rst

```diff
@@ -0,0 +1,376 @@
+.. _AdvancedExampleDeviatedPoroElasticWellbore:
+
+
+####################################################
+Deviated Poro-Elastic Wellbore Problem
+####################################################
+
+------------------------------------------------------------------
+Problem description
+------------------------------------------------------------------
+
+This example aims to solve a typical injection problem of a deviated poroelastic open wellbore subjected to a fluid pressure injected inside the hole. The problem geometry is generated by the interal wellbore mesh generator. The poroelastic coupled solver, that is a combination of the solid mechanic solver and the single phase flow solver, is employed to solver this coupling problem. In-situ stress and pressure are excluded from this example.
+
+Analytical results of the fluid pressure, and the radial and hoop stresses around the wellbore are expressed in the Laplace space as `(Detournay and Cheng, 1988) <https://www.sciencedirect.com/science/article/abs/pii/0148906288922991>`__ :
+
+.. math::
+    p = p_0 \frac{kn(0,R \sqrt{s})}{s kn(0,\sqrt{s})}
+
+.. math::
+    \sigma_{rr} = -b \frac{1-2\nu}{1-\nu} p_0 \frac{-R * kn(1,R \sqrt{s}) + kn(1,\sqrt{s})}{R^2 \sqrt{s^3} kn(0,\sqrt{s})}
+
+.. math::
+    \sigma_{\theta\theta} = -b \frac{1-2\nu}{1-\nu} p - \sigma_{rr}
+
+where :math:`s` is the Laplace variable, :math:`R` is the dimensionless radial coordinate that is defined by the radial coordinate normalized by the hole radius, :math:`\nu` the Poisson ratio and :math:`b` the Biot coefficient. Fluid pressure and stresses in time space are obtained from these analytical expression by the inverse Laplace transformation (see the attached Python script for more details).  
```

Comment:
```suggestion
where :math:`s` is the Laplace variable, :math:`kn` is the is the modified Bessel function of second kind of order n, :math:`R` is the dimensionless radial coordinate that is defined by the radial coordinate normalized by the wellbore radius, :math:`\nu` is the Poisson ratio and :math:`b` is the Biot coefficient. Fluid pressure and stresses in time space are obtained from these analytical expressions by inversing Laplace transformation (see the attached Python script for more details).  
```

Reply:
Please verify the analytical solution. Here, `s` should be an expression of Laplace variable and diffusion coefficient. This analytical solutions are expressed in the local coordinate system (along wellbore trajectory). I suggest to make a note here.

Reply:
Done